mindsdb/interfaces/datastore/datastore.py

```diff
@@ -173,6 +173,8 @@ def get_data(self, name, where=None, limit=None, offset=None, company_id=None):
         offset = 0 if offset is None else offset
         ds = self.get_datasource_obj(name, company_id=company_id)
 
+        if ds is None:
+            raise Exception(f'Unknown file: {name}')
```

Comment:
1. That will be 'file' only in case of 'file' integration.
2. '{name}' in general case won't give to the user any info: we don't show dataset name anywhere

Reply:
removed this change